backend/typescript/types.ts

```diff
@@ -70,15 +84,12 @@ export type CampDTO = {
   campSessions: string[];
 };
 
-export type CampSessionDTO = {
-  id: string;
-  camp: string;
-  campers: string[];
-  waitlist: string[];
-  dates: string[];
-  startTime: string;
-  endTime: string;
-  active: boolean;
+export type GetCampDTO = Omit<CampDTO, "campSessions" | "formQuestions"> & {
+  formQuestions: FormQuestionDTO[];
+  campSessions: (Omit<
+    CampSessionDTO,
+    "id" | "camp" | "campers" | "waitlist"
+  > & { registerations: number; waitlists: number })[];
```

Comment:
think this naming makes it a bit clearer, what do you thinkk 
```suggestion
  > & { numCampers: number; numWaitlist: number })[];
```